tests/test_schema.py

```diff
@@ -30,13 +30,57 @@ def load_json(filename):
     return doc
 
 
+def test_delivery():
+    """Test delivery superset schema."""
+    upsert = load_json('valid.json')
+    takedown = copy.deepcopy(upsert)
+    takedown['action'] = 'takedown'
+    assert schema.validate_schema(schema.delivery, upsert) == upsert
+    assert schema.validate_schema(schema.delivery, takedown) == takedown
+
+
 def test_empty_document():
     """Test that an empty document doesn't validate."""
     doc = load_json('empty.json')
     with pytest.raises(schema.MultipleInvalid):
         schema.track_bundle(doc)
 
 
+@pytest.mark.parametrize('type_', (
+    'AdvertsiementSupportedModel',
+    'DevicefeeModel',
+    'PayAsUGoModel',
+    'Rights',
+    '$5',
+))
+def test_invalid_commercial_model_type(type_):
+    with pytest.raises(schema.CommercialModelTypeInvalid):
```

Comment:
When working with a known set of values (e.g., `COMMERCIAL_MODEL_TYPES`), it's often a good idea to assert that the invalid value isn't one of the values. If a bug is later introduced to `schema.CommercialModelType` that causes it to stop raising `CommercialModelTypeInvalid`, it's easy to think that your test is suddenly failing because the set of values has changed.